ospf6d/ospf6_zebra.c

```diff
@@ -176,12 +176,24 @@ static int ospf6_zebra_if_address_update_delete(ZAPI_CALLBACK_ARGS)
 	return 0;
 }
 
+static int is_prefix_default(struct prefix_ipv6 *p)
+{
+	struct prefix_ipv6 q;
```

Comment:
`memset` could be replaced with `struct prefix_ipv6 q = {}` here.

Reply:
Fixed